docs/userguide/constraints.md

```diff
@@ -315,10 +315,13 @@ hence the `SearchSpace` should effectively only contain one of them.
 Complex properties such as permutation invariance not only affect the search space but
 should ideally also constrain the surrogate model. For instance, the kernels in a
 Gaussian process can be made permutation-invariant to reflect this constraint, which
-generally results in a better learning curve. Note, however, that not all surrogate
-models provided by BayBE can easily include such invariances. In these cases, the
-invariance is ignored during model fitting, which will not be an issue, but just means these models
-cannot benefit from a priori known constraints and invariances between parameters.
+generally results in a better learning curve. Note that at this stage no
+surrogate model provided by BayBE takes care of these invariances. This means the
+invariance is ignored during model fitting. This is not a big issue, but just means 
```

Comment:
The "this is not a big issue" feels too colloquial for me, we could just not write it and start with "This just means"...

Reply:
u gucci with this?
`This just means these models cannot benefit from a priori known constraints and invariances between 
parameters; generally, the optimization will still work. `

Reply:
Muito gucci.